### PR TITLE
Edit BIRS funding proposal after Board Meeting

### PR DESCRIPTION
**Pull request by santisoler**
Changes:

* Rephrase implementation paragraph: the written output will be produced by the Workshop as a whole and not necessarily by the beneficiaries.
* Remove unwanted list inherited from the template.
* Change the budget estimations to CAD.
* Clarify that the required amount is the maximum.
* Add an update on third-party funding denials.

cc @kwinkunks @rowanc1

### CODE REVIEW

File: proposals/02_birs-financial-support.md [hunk 1]
```diff
@@ -94,58 +94,48 @@ expenses.
...
-| **Total**     | **1515.00** |
+| Meal expenses |    420.00   |
+| Travel costs  |   1320.00   |
+| Swag          |    260.00   |
+| **Total**     | **2000.00** |
 
-Through this proposal we would like to ask Software Underground to provide
-**1515.00 USD** to cover meal, travel and (optionally) swag expenses for the
+Through this proposal we would like to ask Software Underground to provide up
+to **2000.00 CAD** to cover meal, travel and (optionally) swag expenses for the
```
Comment (rowanc1) [inline proposals/02_birs-financial-support.md:137]: Thanks for changing to CAD! 🚀

File: proposals/02_birs-financial-support.md [hunk 2]
```diff
@@ -164,19 +154,25 @@ the event, social media publication and blog post.
...
 the event organizers and Software Underground Board Members.
 
-After the event, beneficiaries will be encouraged to write some blog post
-commenting their experience on the workshop.
-These blog posts will have to be licensed under a Creative Commons Attribution
-license and will be published on Software Underground website and/or the
-Workshop's website.
-Beneficiaries can choose to remain anonymous in these blog posts.
+After the event, the Workshop will output some written products in a form of
+blog posts, tutorials, or similar.
```
Comment (rowanc1) [inline proposals/02_birs-financial-support.md:158]: I think much better now that this is on the event rather than the individuals.

**Review by rowanc1 — APPROVED** [review]
Looks good -- glad that SWUNG can help out here!

File: proposals/02_birs-financial-support.md (hunk 1 above)
Comment (santisoler) [reply proposals/02_birs-financial-support.md:137]: 🍁

**Pull request merged**